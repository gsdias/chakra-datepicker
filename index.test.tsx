import React from 'react'
import renderer from 'react-test-renderer'
import ThemeProvider from '../../providers/ThemeProvider'
import Calendario from '.'

describe('Calendario', () => {
  it('Should render correctly', () => {
    const tree = renderer
      .create(
        <ThemeProvider>
          <Calendario />
        </ThemeProvider>,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
