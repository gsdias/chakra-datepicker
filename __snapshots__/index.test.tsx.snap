// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendario Should render correctly for Avatar Icon 1`] = `
Array [
  <input
    className="chakra-input css-1c6j008"
    onBlur={[Function]}
    onChange={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    placeholder="Date"
    readOnly={false}
    value=""
  />,
  <span />,
]
`;
